src/Microsoft.Performance.Toolkit.Engine.Tests/ToolkitEngineCreateInfoTests.cs

```diff
@@ -17,6 +29,8 @@ public void DefaultRuntime_Set()
 
             string expectedName = typeof(EngineCreateInfo).Assembly.GetName().Name;
             Assert.AreEqual(expectedName, sut.RuntimeName);
+            Assert.IsNotNull(sut.OptionsResolver, "Options Resolver is null when a default is expected");
```

Comment:
Is there a reason to put this here instead of in its own unit test? The test this was added to is named "DefaultRuntime_Set," so I would expect it to _only_ fail if the default runtime is not set.

Reply:
Haha, I didn't understand why the test was named that way. Going to make my own unit tests in EngineCreateInfo and Engine now.